setup/symfony_server.rst

```diff
@@ -0,0 +1,271 @@
+Symfony Local Web Server
+========================
+
+You can run Symfony applications with any web server (Apache, nginx, the
+internal PHP web server, etc.) However, Symfony provides its own web server to
+make you more productive while developing your apps.
+
+This server is not intended for production use and it supports HTTP/2, TLS/SSL,
+automatic generation of security certificates, local domains, and many other
+features that sooner or later you'll need when developing web projects.
+
+Installation
+------------
+
+The Symfony server is distributed as a free installable binary without any
+dependency and support for Linux, macOS and Windows. Go to `symfony.com/download`_
+and follow the instructions for your operating system.
+
+Getting Started
+---------------
+
+The Symfony web server is started once per project, so you may end up with
+several instances (each of them listening to a different port). This is the
+common workflow to serve a Symfony project:
+
+.. code-block:: terminal
+
+    $ cd my-project/
+    $ symfony server:start
+
+      [OK] Web server listening on http://127.0.0.1:....
+      ...
+
+    # Now, browse the given URL in your browser, or run this command:
+    $ symfony open:local
+
+Running the server this way makes it display the log messages in the console, so
+you can't run other commands. If you prefer, you can run the Symfony server in
```

Comment:
```suggestion
you won't be able to run other commands at the same time. If you prefer, you can run the Symfony server in
```

Reply:
I'm not questioning your fix because you are native in English and I'm not ... but this is confusing to me --> *"... at the same time."*

To me that means you can't run commands **simulataneously** ... but I don't want that. I may (or may not) run commands **later**, but if the terminal is blocked by Symfony's server, I can't.

So, it's still correct to add *"... at the same time."*  ? Thanks!

Reply:
It should still be correct - 'at the same time' doesn't necessarily have to mean that you invoke them simutaneously, it just means that there is some concurrent activity.

Reply:
Fantastic! Thanks for fixing this ... and for teaching me about this expression!